product_sticker/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2023 Moduon Team S.L. <info@moduon.team>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Product Sticker",
+    "version": "16.0.1.0.0",
+    "author": "Moduon, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "website": "https://github.com/OCA/product-attribute",
+    "category": "Sales Management",
+    "depends": [
+        "product",
+    ],
+    "data": [
+        "security/ir.model.access.csv",
+        "views/product_sticker_views.xml",
+        "views/product_attribute_views.xml",
+        "views/product_template_views.xml",
+        "data/menus.xml",
+    ],
+    "maintainers": ["Shide", "rafaelbn"],
```

Comment:
why did you add @rafaelbn on the migration? You were already a maintainer :thinking:

Reply:
I develop this module with @rafaelbn , under the umbrella of @moduon, so If I leave Moduon, someone needs to maintain this module and he is the right person to request improvements or approve changes.

I can do it in a separate PR when it's merged because I will be the maintainer in v16, but doesn't make sense to me if I'm the maintainer of the module right now and I consider acceptable to add @rafaelbn as a maintainer too.